### ISSUE TITLE
C++ completions with ccls provides incorrect results

### ISSUE THREAD

**Issue by srinathava** [prabirshrestha/vim-lsp#544]
Hi,

I am trying out vim-lsp using the provided minimal.vimrc with CCLS server. I have a simple program:

```
struct Foobar {
        int f1;
        int f2;
};

int main() {
        Foobar x;
}
```

When I try to do auto-completion by typing "x." right after the line "Foobar x;", I see a menu popup with the items:

```
f1 : int field Foobar
f2 : int field Foobar
```
If I try to press `<CTRL-N>` to choose f1, I actually get:

```
x.f1 : int
```
on the line instead of just

```
x.f1
```

Attached are the logs for vim-lsp.log and asyncomplete.log. A preliminary look at the logs suggests that vim-lsp should be using the "filteredText" instead of "label" from the LSP server.

Thanks,
Srinath

[vim-lsp.log](https://github.com/prabirshrestha/vim-lsp/files/3748577/vim-lsp.log)
[asyncomplete.log](https://github.com/prabirshrestha/vim-lsp/files/3748578/asyncomplete.log)

**Comment by mkwork** [comment]
I have the similar problem since some update.

**Comment by ruifm** [comment]
Same problem. I had to deactivate ccls and only use clangd and cquery.

**Comment by bfrg** [comment]
That's a known issue, see #328.

**Comment by bfrg** [comment]
For a solution, see following [comment](https://github.com/prabirshrestha/vim-lsp/issues/328#issuecomment-555304328).

**Comment by hrsh7th** [comment]
I think this issue fixed by #688.
Could someone test with latest vim-lsp?
